### PR TITLE
Feat/levels

### PR DESCRIPTION
**Pull request by bendsp**

### CODE REVIEW

File: engine/includes/components.hpp [hunk 1]
```diff
@@ -114,6 +114,10 @@ struct Player{
     Player(int id) : id(id) {}
 };
 
+struct Npc {
+    //TODO
+};
```
Comment (PaulThblt) [inline engine/includes/components.hpp:119]: To do as in used but not defined or future feature?

Comment (bendsp) [reply engine/includes/components.hpp:119]: Future use

File: server/includes/levels.hpp [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/**
+ * @file levels.hpp
+ * @brief 
+ * @date 08-01-2024
+ */
+
+#pragma once
+
+#include "../../engine/includes/ECS.hpp"
+#include "../../engine/includes/components.hpp"
```
Comment (PaulThblt) [inline server/includes/levels.hpp:10]: `components.hpp` already included in `ECS.hpp`
```suggestion
#include "../../engine/includes/ECS.hpp"

```

File: server/includes/parser.hpp
Comment (PaulThblt) [inline server/includes/parser.hpp:1]: Do we need to keep the comment?

File: server/src/parser.cpp
Comment (PaulThblt) [inline server/src/parser.cpp:1]: Do we need to keep the comment?

**Review by PaulThblt — APPROVED** [review]

**Pull request merged**